src/hooks/useAPI.ts

```diff
@@ -253,6 +255,44 @@ const useAPI = (version: ApiVersion = "v1") => {
     [client, walletAddress],
   );
 
+  const getPlay3List = useCallback(async () => {
+    const contractAddress = contractAddresses[chainName]?.play3List;
+    if (isLoading) return;
+    if (!client || !contractAddress) {
+      return undefined;
+    }
+    let res: WhiteList = [];
+    let i = 0;
+
+    while (true) {
+      const queryData = getQueryData({
+        get_complete_meme_data_list: {
+          start_after: "",
```

Comment:
The first is "", but after that it must be the last cont_addr of the result value.